runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -70,6 +126,82 @@ public void updateClanChatTitle()
 		}
 	}
 
+	private boolean isClanChatClosed()
+	{
+		Widget widget = client.getWidget(WidgetInfo.CLAN_CHAT);
+
+		return widget == null || widget.isHidden();
+	}
+
+	@Override
+	public boolean onChatboxInput(ChatboxInput chatboxInput)
+	{
+		if (!config.swapChat() && (!config.mobileChat() || isClanChatClosed()))
+		{
+			return false;
+		}
+
+		String msg = chatboxInput.getValue();
+
+		if (msg.startsWith("/"))
+		{
+			msg = msg.substring(1);
+			chatboxInput.setChatType(ChatMessageType.PUBLIC_MODE.getType());
+		}
+		else
+		{
+			msg = "/" + msg;
+			chatboxInput.setChatType(ChatMessageType.CLANCHAT_MODE.getType());
+		}
+
+		chatboxInput.setValue(msg);
+
+		executor.execute(() ->
+		{
+			chatboxInput.resume();
+		});
+
+		return true;
+	}
+
+	@Override
+	public boolean onPrivateMessageInput(PrivateMessageInput privateMessageInput)
+	{
+		return false;
+	}
+
+
```

Comment:
Remove this empty line

Reply:
This empty line is still here

Reply:
I removed it again. And again.

Reply:
if you click View for the file, it's gone